src/tools/ScenarioMeasurement/Startup.Tests/StartupTests.cs

```diff
@@ -0,0 +1,90 @@
+using Microsoft.DotNet.PlatformAbstractions;
+using Reporting;
+using ScenarioMeasurement;
+using System;
+using System.IO;
+using System.Resources;
+using Xunit;
+
+namespace Startup.Tests
+{
+    public class StartupTests
```

Comment:
We talked about tests to validate that the dictionaries representing the keyword maps were properly filled out. Did you think about that?

Reply:
Almost forgot. Will add the validation to this PR.